Gemfile

```diff
@@ -1,14 +1,14 @@
 source 'https://rubygems.org'
 
-gem 'jekyll', '~> 3.9.2'
-gem 'kramdown', '~> 2.3.2'
```

Comment:
@bellapequette were you unable to build with this tilde in place?

Reply:
@nglaser3 identified an issue with the "Jekyll serve" command and helped me resolve it by removing the tilde. It was unable to be built with the tilde in place.